[FEATURE] Support for XLIFF 2.0 translation files (#6113)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Faq.rst

..  _Localization Team: https://typo3.org/community/teams/localization

.  _crowdin-faq-xliff-version:

Why do XLIFF files on Crowdin look different now?
-------------------------------------------------

TYPO3 14 and newer use the modern **XLIFF 2.0** format for translation files.
This version introduces a cleaner structure with `<unit>` and `<segment>`
elements and uses the `<target state="…">` attribute instead of the older
`approved="yes"` attribute used in XLIFF 1.2.

Crowdin supports both formats, and TYPO3 automatically detects which version is
used — so you do not need to convert files manually.

..  seealso::

>     Learn more about XLIFF 2.0 in TYPO3:

Reviewer: ```suggestion
    Learn more about XLIFF 2.x in TYPO3:
```

    :ref:`xliff`

..  _crowdin-faq-xliff-convert:

> Do I need to convert existing XLIFF 1.2 files to 2.0?

Reviewer: ```suggestion
Do I need to convert existing XLIFF 1.2 files to 2.x?
```

-----------------------------------------------------

Not necessarily.

-  As long as your extension **still supports TYPO3 13 LTS**, you **must not**

>    switch to XLIFF 2.0, because TYPO3 13 only supports XLIFF 1.2.

Reviewer: ```suggestion
   switch to XLIFF 2.x, because TYPO3 v13 only supports XLIFF 1.2.
```

> -  TYPO3 14 and later can read both XLIFF 1.2 and 2.0 files seamlessly.

Reviewer: ```suggestion
-  TYPO3 v14 and later can read both XLIFF 1.2 and 2.x files seamlessly.
```

> -  For **new extensions** that target TYPO3 14 and above, use **XLIFF 2.0**

Reviewer: ```suggestion
-  For **new extensions** that target TYPO3 14 and above, use **XLIFF 2.x**
```
